src/components/FileTreeNode/index.spec.tsx

```diff
@@ -443,4 +504,105 @@ describe(__filename, () => {
       expect(type).toEqual(messages[0].type);
     });
   });
+
+  describe('isKnownLibrary', () => {
+    it('returns false when there is no linter message', () => {
+      const path = 'jquery.js';
+      const messages = [] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(false);
+    });
+
+    it('returns false when there is more than one global message', () => {
+      const path = 'jquery.js';
+      const messages = [
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          id: [LINTER_KNOWN_LIBRARY_CODE],
+          line: null,
+          type: 'notice',
+        },
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          line: null,
+          type: 'notice',
+        },
+      ] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(false);
+    });
+
+    it('returns false when there are inlined linter messages', () => {
+      const path = 'jquery.js';
+      const messages = [
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          id: [LINTER_KNOWN_LIBRARY_CODE],
+          line: null,
+          type: 'notice',
+        },
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          line: 123,
+          type: 'notice',
+        },
+      ] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(false);
+    });
+
+    it('returns false when there are only inlined linter messages', () => {
+      const path = 'jquery.js';
+      const messages = [
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          line: 123,
+          type: 'notice',
+        },
+      ] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(false);
+    });
+
+    it('returns false when a path is not a known library', () => {
+      const path = 'jquery.js';
+      const messages = [
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          id: ['another_code'],
+          line: null,
+          type: 'notice',
+        },
+      ] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(false);
+    });
+
+    it('returns true when a path is a known library', () => {
+      const path = 'jquery.js';
+      const messages = [
+        {
+          ...fakeExternalLinterMessage,
+          file: path,
+          id: [LINTER_KNOWN_LIBRARY_CODE],
+          line: null,
+          type: 'notice',
+        },
+      ] as ExternalLinterMessage[];
+      const map = _getMessageMap(messages);
+
+      expect(isKnownLibrary(map, path)).toEqual(true);
```

Comment:
It would be nice to see an additional test for when a known library code exists but for the wrong path